mitodl/open-discussions
# Add sleep to transcript job

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2445

#### What's this PR do?
This pr adds sleeps to the video transcripts job to avoid 429 errors from youtube.

#### How should this be manually tested?
Run 
```
from course_catalog.tasks import *
get_youtube_transcripts()
```
to ensure the task still works 
#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
